src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc;
+
+import static io.debezium.config.Field.Group.CONNECTOR_ADVANCED;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiConsumer;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.errors.ConnectException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.ConfigDefinition;
+import io.debezium.config.EnumeratedValue;
+import io.debezium.config.Field;
+
+public class JdbcSinkConnectorConfig {
+    private static final Logger LOGGER = LoggerFactory.getLogger(JdbcSinkConnectorTask.class);
+
+    public static final String CONNECTION_URL = "connection.url";
+    public static final String CONNECTION_USER = "connection.username";
+    public static final String CONNECTION_PASSWORD = "connection.password";
+    public static final String CONNECTION_POOL_MIN_SIZE = "connection.pool.min_size";
+    public static final String CONNECTION_POOL_MAX_SIZE = "connection.pool.max_size";
+    public static final String CONNECTION_POOL_ACQUIRE_INCREMENT = "connection.pool.acquire_increment";
+    public static final String CONNECTION_POOL_TIMEOUT = "connection.pool.timeout";
+    public static final String PK_MODE = "pk.mode";
+    public static final String PK_FIELDS_REGEX = "pk.field.*";
+
+    private final io.debezium.config.Configuration config;
+    private final PKMode pkModeValue;
+    private final Map<String, List<String>> pkModeMap = new HashMap<>();
+
+    /**
+     * The set of predefined PKMode options or aliases.
+     */
+    public static enum PKMode implements EnumeratedValue {
```

Comment:
Is that taken from somewhere else by any chance?

Reply:
Yep, but Im gonna delete it for this PR